testsuite/features/reposync/allcli_update_activationkeys.feature

```diff
@@ -125,10 +125,11 @@ Feature: Update activation keys
     And I wait for child channels to appear
     And I select the parent channel for the "proxy_container" from "selectedBaseChannel"
     And I wait for child channels to appear
-    And I include the recommended child channels
-    And I wait until "SLE-Manager-Tools-For-Micro5-Pool for x86_64 5.5" has been checked
-    And I check "SUSE-Manager-Proxy-5.0-Pool for x86_64"
-    And I check "SUSE-Manager-Proxy-5.0-Updates for x86_64"
+    # TODO: WORKAROUND: When they are ready add them again
+    # And I include the recommended child channels
+    # And I wait until "SUSE Multi-Linux Manager Client Tools for SL Micro 6 x86_64" has been checked
+    And I check "SUSE-Multi-Linux-Manager-Proxy-5.1 for x86_64"
+    And I check "SUSE-Multi-Linux-Manager-Retail-Branch-Server-5.1 for x86_64"
```

Comment:
just for understanding, if this scenario is about the proxy, why you also need RBS?

Reply:
We use the same key for both retail and proxy and we add both channels to use. I am not sure that this is correct logically yes but for now let's say we stay on the same as 5.0 and I will discuss this with QE and see if we want to change this.

Reply:
I made a note for me to check on this, let's merge as is for now